ch.tlaun.TL.yaml

```diff
@@ -0,0 +1,86 @@
+app-id: ch.tlaun.TL
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+  - name: flite
+    config-opts:
+      - --enable-shared
+    sources:
+      - type: git
+        url: https://github.com/festvox/flite.git
+        tag: v2.2
+        commit: e9e2e37c329dbe98bfeb27a1828ef9a71fa84f88
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([.\d]+)$
+  - name: xrandr
+    sources:
+      - type: archive
+        url: https://www.x.org/archive/individual/app/xrandr-1.5.1.tar.gz
+        sha256: 7b99edb7970a1365eaf5bcaf552144e4dfc3ccf510c4abc08569849929fb366e
+  - name: bootstrap
+    buildsystem: simple
+    sources:
+      - type: file
+        path: ch.tlaun.TL
+        dest: bin
+      - type: file
+        dest: tl
+        url: https://tlauncherrepo.com/legacy/bootstrap/00aa8d0014ce30f2116a13b8313de65cc9be690f475758d83359e7756fa848f0.jar
+        sha256: 00aa8d0014ce30f2116a13b8313de65cc9be690f475758d83359e7756fa848f0
+        dest-filename: bootstrap.jar
+        x-checker-data:
+          type: json
+          url: https://tlauncherrepo.com/brands/legacy/bootstrap.json
+          version-query: .update.bootstrap.version
+          url-query: .update.bootstrap.downloads.JAR.url[0]
+    build-commands:
+      - mv bin/ch.tlaun.TL /app/bin
+      - mv tl /app/lib
+  - name: resources
+    buildsystem: simple
+    sources:
+      - type: file
+        path: ch.tlaun.TL.desktop
+      - type: file
+        path: ch.tlaun.TL.appdata.xml
+      - type: file
+        path: ch.tlaun.TL.48x48.png
+      - type: file
+        path: ch.tlaun.TL.64x64.png
+      - type: file
+        path: ch.tlaun.TL.96x96.png
+      - type: file
+        path: ch.tlaun.TL.128x128.png
+      - type: file
+        path: ch.tlaun.TL.192x192.png
+      - type: file
+        path: ch.tlaun.TL.256x256.png
+    build-commands:
+      - install -Dt /app/share/applications ch.tlaun.TL.desktop
+      - install -Dt /app/share/metainfo ch.tlaun.TL.appdata.xml
+      - install -D ch.tlaun.TL.48x48.png /app/share/icons/hicolor/48x48/apps/ch.tlaun.TL.png
+      - install -D ch.tlaun.TL.64x64.png /app/share/icons/hicolor/64x64/apps/ch.tlaun.TL.png
+      - install -D ch.tlaun.TL.96x96.png /app/share/icons/hicolor/96x96/apps/ch.tlaun.TL.png
+      - install -D ch.tlaun.TL.128x128.png /app/share/icons/hicolor/128x128/apps/ch.tlaun.TL.png
```

Comment:
You only need 128x128. 256x256 is ok too. The rest is unnecessary. A SVG is a good idea if there is one.

Reply:
fixed